release/mixer.sh

```diff
@@ -22,7 +22,6 @@ var_load_all
 
 IS_UPSTREAM=${IS_UPSTREAM:-false}
 NUM_DELTA_BUILDS=${NUM_DELTA_BUILDS:-10}
-MIXER_OPTS=${MIXER_OPTS:-"--native"}
```

Comment:
```
MIXER_OPTS=${MIXER_OPTS:-""}
```
This makes our lives easier to maintain the --offline

Reply:
IOW dont remove the capability to pass an option, just update the default.

Reply:
For parameters that can be supplied, are do we want to create an explicit list of defaults at the top of our scripts?  ${MIXER_OPTS} will expand to nothing and can still be passed even without this at the top of the file.

Reply:
That is the idea behind having everything in the top of the file. Make it clear which are the options and what are the default values

Reply:
Either that or a line of documentation in the top of the file, and then we can just remove this line

Reply:
Sounds good!

Reply:
I ran with that idea a little and formalized the interface a bit more for all of the parameters to clean that up a little.